imdown-mobile/package.json

```diff
@@ -15,6 +15,7 @@
     "@react-navigation/native": "^5.0.5",
     "@react-navigation/stack": "^5.0.5",
     "emoji-picker-react": "^3.2.1",
+    "core-js": "^3",
```

Comment:
I would drop this commit in an interactive rebase and then pull changes from master using `git rebase origin master`